### PR TITLE
[RFR] Bootstrap

### PR DESCRIPTION
**Pull request by Kmaschta**
- [x] ~~Project bootstrap with a runnable android app~~
- [x] ~~.travis.yml with one passing test~~
- [x] ~~ES6~~

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# OSX
+#
+.DS_Store
```
Comment (fzaninotto) [inline .gitignore]: nah, please add here only files generated by the project that you don't want committed. `.DS_Store` should go into the *user* configuration, not the *project* configuration. In other terms, it's not your problem.

File: index.android.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/**
+ * Sample React Native App
```
Comment (fzaninotto) [inline index.android.js]: Don't forget to remove that one

File: .gitignore (hunk 1 above)
Comment (Kmaschta) [inline .gitignore]: The file was automatically generated by react-native, but (again) you're
right !

**Comment by Kmaschta** [comment]
@fzaninotto Finally, I can start !

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1,33 @@
+# Xcode
```
Comment (fzaninotto) [inline .gitignore]: You don't need this XCode part

**Pull request merged**